src/main/telemetry/hott.c

```diff
@@ -465,6 +489,7 @@ void handleHoTTTelemetry(void)
 {
     static uint32_t serialTimer;
     uint32_t now = micros();
+    alarmSound_currentTime = now;
```

Comment:
is it possible to avoid the ram usage of `alarmSound_CurrentTime` by passing `now` to 'time_for_Voice_Alarm' ?